framework/platform/bpl/include/bpl/bpl_cfg.h

```diff
@@ -490,6 +491,14 @@ int cfg_get_hostap_iface(int32_t radio_num, char hostap_iface[BPL_IFNAME_LEN]);
  */
 int cfg_get_all_prplmesh_wifi_interfaces(BPL_WLAN_IFACE *interfaces, int *num_of_interfaces);
 
+/**
+ * @brief Get persistent-db enable.
+ * 
+ * @param [out] enable Enable persistent DB read from config - default value (if parameter is not configured) is DEFAULT_PERSISTENT_DB.
+ * @return true on success, otherwise false.
+ */
```

Comment:
```suggestion
/**
 * @brief Returns whether the persistent DB is enabled.
 * 
 * @param [out] enable true if the DB is enabled and false otherwise.
 * @return true on success, otherwise false.
 */
```

Reply:
accepted your suggestions - thanks